bin/dataproc_template_constants.sh

```diff
@@ -17,3 +17,8 @@
 
 
 JAR_FILE=dataproc-templates-1.0-SNAPSHOT.jar
+#Default job type serverless spark
+JOB_TYPE=serverless
+FAILURE_CODE=127
+SERVERLESS_JOB_CODE=serverless
+DATAPROC_JOB_CODE=dataproc
```

Comment:
Not sure if this is the best naming. A Serverless job is still a Dataproc job. Dataproc Serverless is an extension of Dataproc. 

For this it might not be totally clear for a new reader that by DATAPROC_JOB we mean a job that runs in a Dataproc cluster rather than one running in Dataproc Serverless. I am open to debate here.